Sources/SwerverCore/Response/HttpResponse.swift

```diff
@@ -0,0 +1,64 @@
+import Foundation
+
+public class HttpResponse {
+    
+    public static let LINE_SEPARATOR: String = "\r\n"
+    public static let SPACE: String = " "
+    public static let HTTP_VERSION: String = "HTTP/1.1"
```

Comment:
You are asking for the HTTP version in a class above. Why is "HTTP/1.1" hard-coded here?

Reply:
the HTTP/1.1 is important for constructing a valid response. I tried leaving it out and I'll get an invalid response error from the client. so I chose to leave it in the response class as a class constant